Updated Molecules documentation. (#606)
StadGent/drupal_theme_gent-base: styleguide/components/31-molecules/gent-info/README.md

# Gentinfo

Gentinfo is the central point of contact for information about the city council and the city of Ghent.

## When to use this component

Use the Gentinfo component to show Gentinfo contact information as a call-to-action on a web page when it is relevant to the page and when a separate call-to-action for Gentinfo is useful for users.

> The Gentinfo component should always be used **as is**. The elements, style, layout and content in the Gentinfo component **cannot be changed in anyway**.

Reviewer: 😇

```suggestion
The Gentinfo component should always be used **as is**. The elements, style, layout and content in the Gentinfo component **cannot be changed in any way**.
```

In the examples the Gentinfo component is shown as it should be included in different languages.

## When not to use this component

* Do not use the Gentinfo component to present other content than the Gentinfo contact information.

> * Do not use the Gentinfo component to present other call-to-actions. For other call-to-actions, use the call-to-action component.

Reviewer: ```suggestion
* Do not use the Gentinfo component to present other call-to-actions. For other call-to-actions, use the <a href="{{path './cta-block.html'}}">call-to-action</a> component.
```

Author: @hstandaert The reason that I did not a link here (yet) is because the Call to action component needs to be refactored (Contact blok and Call to action become one). Because I was not sure what the URL would be, I did not put a link. Will it be this URL? Otherwise, maybe, we could add a TO DO comment?

Reviewer: That makes sense! A todo comment is perfect 👍 
```suggestion
<!-- @TODO  Link to the cta component -->
* Do not use the Gentinfo component to present other call-to-actions. For other call-to-actions, use the call-to-action component.
```
